lib/note-content-editor.tsx

```diff
@@ -74,6 +75,7 @@ class NoteContentEditor extends Component<Props> {
 
   state: OwnState = {
     content: '',
+    contentDiv: createRef(),
     editor: 'fast',
```

Comment:
@dmsnell should it look like this? https://github.com/Automattic/simplenote-electron/pull/2298/commits/535393d9dc11ffe844c4be2d465cfc2eec7ee4fc

TypeScript makes a sadface about how `Property 'contentDiv' does not exist on type 'NoteContentEditor'`.

Reply:
we should create the `createRef()` before `componentDidMount()`. it's going to be easiest to declare it on the class instance even outside the constructor function. we can remember here that React is creating a class (which initializes the JavaScript class) and _then_ calling `componentDidMount()` only _after_ the actual virtual DOM nodes have been returned from `render()`. since we are passing that ref into `render()` it should be there already

```ts
class NoteContentEditor extends Component<Props> {
	bootTimer… = null;
	editor: Editor.IStandaloneCodeEditor | null = null;
	…
	contentDiv = createRef<HTMLDivElement>();

	state: OwnState = {
		…
	}

	…
}
```

note that TypeScript will infer the type of `contentDiv` for us and we won't have to import `RefObject`